build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/page_config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "disabledExtensions": {
+      "@jupyterlab/apputils-extension:announcements": true,
+      "@amzn/sagemaker-jupyterlab-emr-extension": true,
```

Comment:
Can you explain why each of these extensions needs to be disabled in this entry point?

Reply:
For example: scheduler might be useful

Reply:
The announcements is for fetching JupyterNewsFeed. https://jupyterlab.readthedocs.io/en/stable/user/announcements.html Disabling here for as not helpful.

Sagemaker extensions are not applicable here, e.g. connecting to EMR clusters.

JupyterScheduler will not work out of the box as well because our JupyterLab is short-lived and need to connect to remote compute engine headlessly, we are looking for implementing scheduling by other means.